ProcessesApi/V1/Boundary/Request/Validation/CreateProcessQueryValidator.cs

```diff
@@ -0,0 +1,23 @@
+using FluentValidation;
+using Hackney.Core.Validation;
+using ProcessesApi.V1.Domain;
+using ProcessesApi.V1.Boundary.Constants;
+using System;
+
+namespace ProcessesApi.V1.Boundary.Request.Validation
+{
+    public class CreateProcessQueryValidator : AbstractValidator<CreateProcessQuery>
+    {
+        public CreateProcessQueryValidator()
+        {
+            RuleFor(x => x.TargetId).NotNull()
+                              .NotEqual(Guid.Empty);
+            RuleForEach(x => x.RelatedEntities).NotNull()
+                                        .NotEqual(Guid.Empty);
+            RuleFor(x => x.ProcessName).NotXssString()
```

Comment:
Question: Shouldn't the ProcessName have a value (i.e. not null and not empty)?

Reply:
Probably..